packages/components/src/confirm/component.tsx

```diff
@@ -0,0 +1,80 @@
+// @ts-nocheck (for now)
```

Comment:
Just noting that this comment should eventually be removed

Reply:
👍🏻